CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to the LaunchDarkly Python SDK will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org).
 
+## [2.0.0] - 2016-09-01
+### Multivariate and streaming support
+- Added support for multivariate feature flags. The SDK now supports the following data types for feature flags:
```

Comment:
Try to follow the format here:

http://keepachangelog.com/en/0.3.0/

Sub-section headers for Changed, Added, Removed please.